packages/sdk/src/sdk/api/playlists/PlaylistsApi.ts

```diff
@@ -591,7 +594,8 @@ export class PlaylistsApi extends GeneratedPlaylistsApi {
           await this.storage.uploadFile({
             file: coverArtFile,
             onProgress,
-            template: 'img_square'
+            template: 'img_square',
+            auth: this.auth
```

Comment:
I am surprised the storage service doesn't have direct access to `Auth`? I thought the pattern in SDK was that each service was _created_ with the other services it needs access to.

Reply:
Yeah we should pass auth in the constructor here instead https://github.com/AudiusProject/audius-protocol/blob/main/packages/sdk/src/sdk/sdk.ts#L181

`editFile` does this incorrectly too https://github.com/AudiusProject/audius-protocol/blob/main/packages/sdk/src/sdk/services/Storage/Storage.ts#L53